app/views/posts/index.html.erb

```diff
@@ -4,11 +4,46 @@
   <title>User Posts</title>
 </head>
 <body>
-  <h1>User Posts</h1>
-  <p>User ID: <%= @user.id %></p>
-  <!-- Display user posts here -->
-  <% @posts.each do |post| %>
-    <p><%= post.title %></p>
-  <% end %>
+  <div class="user-container ">
+    <div class="users flex-spaced">
+      <div class="user-photo bordered">Photo</div>
+      <div class="details bordered flex-spaced">
+          <div class="user-name"><%= @user.name %></div>
+          <div class="post-nums">Number of posts: <%= @posts.count %></div>
+      </div>
+    </div>
+    <% @posts.each_with_index do |post, index| %>
+      <div class="flex post-comment">
+        <div class="bio bordered flex-c">
+          <p>post #<%= index + 1 %> </p> 
+          <span><%= post.text %></span>
+        </div>
+        <div class="comment-likes flex gap-1">
```

Comment:
- You have done well so far, but according to the project requirements, this should be a link leading to the `post#show` page. Please fix this 🙏